src/main-process/parse-command-line.js

```diff
@@ -177,6 +193,22 @@ module.exports = function parseCommandLine (processArgs) {
   }
 }
 
+function readProjectSpecificationSync (filepath, executedFrom) {
+  try {
+    const readPath = path.isAbsolute(filepath) ? filepath : path.join(executedFrom, filepath)
+    const contents = CSON.readFileSync(readPath)
```

Comment:
can the `try`/`catch` be moved to just wrap the `readFileSync` call?

Reply:
We want to immediately fall through and error as soon as the file is not found. If the other code is moved outside of the try/catch block, then if the read fails, it is executed before the error is thrown (and may result in another error itself: ie: `cannot read property of undefined`, rather than the error message we want). I could move it outside and add more null checks, but I think  this approach is cleaner.